src/components/Common/DescriptionText.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+class DescriptionText extends Component {
+    render() {
+        return (
+            <div>
```

Comment:
Remove unnecessary identing.
Add a line at end of file (Editors have an option for this).
Don't use div use Grid